### ISSUE TITLE
If a variable is longer than the continuous amount of "#"'s then the code is shifted down, resulting it to not fit in.

### ISSUE THREAD

**Issue by ghost** [artistic-code/JStoART#2]
for example, if we have some normal code in the starting, then there is a long string in console.log, but the string is shorter than the width, it will be shifted down, to where we can fill it in a whole line,  resulting in the code not being able to fit in 
![image](https://user-images.githubusercontent.com/28687771/103259256-935ea780-49be-11eb-8f28-40f1fcb4721e.png)

**Comment by lennardwalter** [comment]
We could also minify globals (add an option for it). Would result in shorter variable names but only works if the file is self contained (packaged with i.e browserify)

**Comment by ghost** [comment]
> We could also minify globals (add an option for it). Would result in shorter variable names but only works if the file is self contained (packaged with i.e browserify)

i dont think this would work, its not the variable names causing this but the long strings.

**Comment by lennardwalter** [comment]
In this case not, but unlike strings, you can't split variable names so a very detailed ascii art with lot's of small holes, etc. would still make problems. So I guess we could just add it. Splitting strings and regexes (#1) is the big problem though so that's what needs to be done
